tests/test_dependencies.py

```diff
@@ -147,6 +147,11 @@ def test_load_save(deps):
     deps2.load(deps_file)
     pd.testing.assert_frame_equal(deps(), deps2())
     os.remove(deps_file)
+    # Wrong extension or file missng
+    with pytest.raises(ValueError, match=r".*'txt'.*"):
```

Comment:
Cool we can use regex here, so it's less critical for test if the error message is slightly changed!